docs/report/report.tex

```diff
@@ -543,6 +541,59 @@ \subsection{Dynamische Tests}
+
+\subsubsection{Automatisierte Tests und Deployment}
+Jeder Pull Request durchlief außerdem mehrere automatisierte Tests - sogenannte \textit{Actions}.
+Diese prüften den Quellcode anhand fest eingestellter Testparameter.
+Durch diese Tests wurde dem Team viel manuelle Arbeit abgenommen und die Fehlerquelle Mensch minimiert.
+Neue Änderungen im Master wurden automatisch auf Github.io veröffentlicht, als auch wurden die \textit{LaTeX} Dateien als PDF im eigenen Dokumente Branch gebaut.
+Für Pull Requests wurde auch die jeweilige Version gebaut und auf einer \colorbox{yellow}{temporären} Webseite veröffentlicht.
+Dank dieser Automatisierungen konnte sich das Team mehr auf das Weiterentwickeln des Projektes konzentrieren.
+
+\subsubsection{Regelmäßige Standups und Retrospektiven}
+Durch die regelmäßigen Standups entstand ein Ort für gegenseitigen Austausch von Problemen und Informationen über Fortschritt und neue Funktionen.
+So wurde verhindert, dass Probleme untergingen, neues Wissen wurde schneller verbreitet und die gesamte Teamproduktivität verbessert.
+Das bei den Retrospektiven entstandene Feedback erwies sich als sehr wichtig für das Teamklima.
+Auch die hierbei angesprochenen Probleme und Verbesserungsvorschläge wurden vom Team als sehr wertvoll für die nächsten Sprints erkannt und entsprechend adaptiert.
+
+\subsubsection{Kommunikation und Eigeninitiative}
+Die Standups hatten auch den Effekt, dass das Team sich anfangs zu sehr auf diesen Kommunikationsweg verließ, wodurch Probleme und wichtige Kommunikation bei Abhängigkeit zwischen Tickets teilweise zu selten stattfanden.
+Tickets wurden anfangs sehr groß gefasst.
+Dadurch entstand der Eindruck, dass diese Tickets sehr komplex und schwierig wären.
+Hier fehlte anfangs die Eigeninitiative, sich aktiv Hilfe zu holen, um diese zu bearbeiten oder die Tickets in kleinere Tickets aufzuteilen.
+Hierbei halfen einerseits die Retrospektiven, in denen dieses Problem angesprochen wurde, sowie vor allem die teaminternen Evaluationen.
+
+\subsubsection{Planung, Zielsetzung und Einschätzung}
+Das größte Problem während des Projektes war, dass gegen Ende eines Sprints geplante Tickets teilweise nicht fertiggestellt wurden.
+Häufig konnten nicht alle Tickets genügend auf Fehler getestet werden und wurden entweder in den nächsten Sprint verschoben oder mit kleineren Bugs in den Master Branch gemerged.
+Hier wurden oftmals Überstunden geleistet und das Team hat hier besonders gelernt, wie wichtig Planung der Tickets sowie das Einschätzen der eigenen Leistung zur Bewältigung verschiedenster Probleme ist.
+So hat das Team sich auch vorgenommen, mindestens zwei Tage vor einem Sprintende alle für den Sprint benötigten Tickets fertiggestellt zu haben.
+Dadurch sollte genügend Zeit geschaffen werden, um Tickets auf Fehlerfreiheit zu testen und konfliktfrei in den Master Branch mergen zu können.
```

Comment:
```suggestion
Dadurch sollte genügend Zeit geschaffen werden, um Tickets auf Korrektheit zu testen und konfliktfrei in den Master Branch mergen zu können.
```

Reply:
umgesetzt in https://github.com/project-leek/project-leek/pull/419/commits/b1990748253786023c795a37fe14470e96e499cd